lib/src/models/multireddit.dart

```diff
@@ -264,13 +282,18 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
   /// [subreddit] is the name of the [Subreddit] to be added to this [Multireddit].
   Future add(/* String, Subreddit */ subreddit) async {
     final subredditName = _subredditNameHelper(subreddit);
+    final newSubredditObject = {'name': "$subredditName"};
+    final encoder = new JsonEncoder.withIndent('  ');
     if (subreddit == null) return;
     final url = apiPath[_kMultiredditUpdate]
         .replaceAll(_userRegExp, _author)
         .replaceAll(_multiredditRegExp, displayName)
         .replaceAll(_subredditRegExp, subreddit);
-    await reddit.put(url, body: {'model': '{"name" : "$subredditName"}'});
-    await refresh();
+    await reddit.put(url,
```

Comment:
Was the previous way you were doing this not working?

Reply:
Unfortunately yeah, as discussed offline, race condition on the Reddit side is likely causing outdated data to be fetched by refresh.